### ISSUE TITLE
ScanObjectNN Results (SageMix+DGCNN)

### ISSUE THREAD

**Issue by sheshap** [mlvlab/SageMix#2]
Hi Authors,

I tried recreating the results mentioned in the paper for the following combinations.

SageMix + DGCNN + SONN_hard ==> 82.5% (paper 83.6%)
SageMix + DGCNN + SONN_easy ==> 86.9% (paper 88.0%).

Please share the configurations used/log files/pre-trained models in your experiments.

I appreciate any help you can provide to recreate the results claimed in the paper.

**Comment by LSanghyeok** [comment]
Hello @sheshap, sorry for the late..
Performance oscillation is a well-known issue in point cloud benchmarks. The results are prone to change due to random seed. 

That's why we provide the table (below) for mean +- std measures on SONN_easy in [openreview](https://openreview.net/forum?id=q-FRENiEP_d&referrer=%5Bthe%20profile%20of%20Sanghyeok%20Lee%5D(%2Fprofile%3Fid%3D~Sanghyeok_Lee1)). 

Method | PointNet | PointNet++ | DGCNN
-- | -- | -- | --
Base | 78.56±0.51 | 86.14±0.39 | 85.72±0.44
+PointMixup | 78.88±0.28 | 87.50±0.26 | 86.26±0.34
+RSMix | 77.6±0.56 | 87.30±0.65 | 85.88±0.59
+**SageMix** | **79.14±0.30** | **88.42±0.26** | **87.32±0.53**

In the case of SONN_hard, we don't have the results for mean +- std measures. Instead, I'm gonna attach the log files for 83.6%.
[logs_SONN_hard.csv](https://github.com/mlvlab/SageMix/files/10457103/logs_SONN_hard.csv)

Sincerely,
Sanghyeok

**Comment by sheshap** [comment]
Dear @LSanghyeok ,

Can you please share the random seed value from your log files?
It will help us reduce the random seed issue and recreate results closer to your paper.
Or please do provide us details on how many times we need to train from scratch to achieve closer results (>88.0).

Thanks Again.
Shivanand

**Comment by RoyJuliet** [comment]
Dear @LSanghyeok ,

I also tried recreating the results mentioned in the paper.
SageMix + DGCNN + SONN_easy ==> 86.05% (paper 88.0%).
Can you please share the model?

Sincerely,
Roy

**Comment by wangxin-fighting** [comment]
@sheshap Hello, may I ask if it is convenient for you to provide the "scanobjectnn" dataset over there? I was unable to successfully download it online and tried downloading it using the "main. py" file as mentioned by the author. In the end, I successfully downloaded a file related to the dataset named "h5_files. zip. 1". However, I am not sure how to handle this dataset in the future. Could you please provide me with some guidance? Thank you in advance for taking the time to review my questions!

**Comment by sheshap** [comment]
@wangxin-fighting 

Please try command below
wget http://hkust-vgd.ust.hk/scanobjectnn/h5_files.zip

**Comment by wangxin-fighting** [comment]
@sheshap Hello, Chief, thank you very much for your previous answer. I have one more question that I would like to ask you again.
I found that the source code provided by the author on the "github" website does not seem to have the pre trained models of dgcnn and pointnet. Do we need to download it ourselves from other places? Another question is how do we view the visualization of point clouds that have been data enhanced after training?
Looking forward to your reply, thank you!

**Comment by sheshap** [comment]
@wangxin-fighting Sorry, I am not the author of the paper. Please check with @LSanghyeok for pre-trained models and visualization.
Wish the best.

**Comment by wangxin-fighting** [comment]
@sheshap Well, thank you very much. Wishing you all the best!
